### ISSUE TITLE
NoClassDefFoundError after upgrade

### ISSUE THREAD

**Issue by homaxto** [damianszczepanik/cucumber-reporting#559]
I have just tried to upgrade from v 2.0.0 to 3.2.0. When running I get the following exception. What am I missing?
I have tried adding log4j 2.7 as dependency with now success.

```
Exception in thread "pool-1-thread-1" java.lang.NoClassDefFoundError: org/apache/logging/log4j/message/ParameterizedMessage
        at org.apache.logging.log4j.message.ParameterizedNoReferenceMessageFactory.newMessage(ParameterizedNoReferenceMessageFactory.java:104)
        at org.apache.logging.log4j.message.AbstractMessageFactory.newMessage(AbstractMessageFactory.java:75)
        at org.apache.logging.log4j.spi.AbstractLogger.logMessage(AbstractLogger.java:2010)
        at org.apache.logging.log4j.spi.AbstractLogger.logIfEnabled(AbstractLogger.java:1884)
        at org.apache.logging.log4j.spi.AbstractLogger.error(AbstractLogger.java:793)
        at org.apache.logging.log4j.core.util.DefaultShutdownCallbackRegistry.run(DefaultShutdownCallbackRegistry.java:76)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.ClassNotFoundException: org.apache.logging.log4j.message.ParameterizedMessage
        at org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy.loadClass(SelfFirstStrategy.java:50)
        at org.codehaus.plexus.classworlds.realm.ClassRealm.unsynchronizedLoadClass(ClassRealm.java:271)
        at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:247)
        at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:239)
        ... 7 more
```

**Comment by damianszczepanik** [comment]
This is rather related to Jenkins than cucumber (as you see there is no stracktrace related to the plugin) but you may find similar issues about it https://github.com/damianszczepanik/cucumber-reporting/issues?utf8=%E2%9C%93&q=log4j and hopefully find solution

**Comment by homaxto** [comment]
Thanks for your reply Damian.
I am not sure what causes this, but Jenkins is not the reason since I was running the test in IntelliJ. Also the only change I made was in maven where I changed the version number of cucumber-reporting from 2.0.0 to 3.2.0. When I change it back everything works again.
If this is not related to cucumber-reporting I think it must be something with the dependencies which are clashing in my project.
